### DISCUSSION TITLE
[Question] How to change the parameter value during runtime?

### DISCUSSION THREAD

**Discussion by tesollo-admin** [isaac-sim/IsaacLab#1591] [Q&A]
Hi, 

When I include the object in a Scene, how do I change the value kinematic_enabled to True after defining the scene cfg?  
Thank you in advance!  

    object = RigidObjectCfg(
        prim_path="/World/envs/env_.*/Object",
        init_state=RigidObjectCfg.InitialStateCfg(pos=(0.0, 0.0, 0.5), rot=(1.0, 0.0, 0.0, 0.0)),
        spawn=sim_utils.UsdFileCfg(
            usd_path=f"./object/object.usd",
            rigid_props=sim_utils.RigidBodyPropertiesCfg(
                kinematic_enabled=False,
                disable_gravity=False,
                enable_gyroscopic_forces=True,
                solver_position_iteration_count=8,
                solver_velocity_iteration_count=0,
                sleep_threshold=0.005,
                stabilization_threshold=0.0025,
                max_depenetration_velocity=10.0,
            ),
            activate_contact_sensors=True,
            mass_props=sim_utils.MassPropertiesCfg(mass=0.188, density=100.0),
        ),
    )

**Comment by shanereetz** [comment]
To clarify, is the intention to create this object with `kinematic_enabled=False` as shown in the snippet above, then set it to `True` later? Or would it work to simply make it `True` from the beginning?

**Comment by RandomOakForest** [comment]
Thanks for posting this. This is most likely not possible. Is there other way to achieve what you need?

**Comment by RandomOakForest** [comment]
Following up on this, the team thinks _PhysX_ under-the-hood of Isaac Lab should be able to handle this fine, so does `omni.physx`, however it may need further work downstream. But on `omni.physx` side you can change the kinematic attribute in runtime and it will toggle the kinematic flag in _PhysX_, thus, this should be fine.

Let us know if you need further help.

**Reply by strn18** [reply]
Hi! Could you point me to the exact `omni.physx` API or function that allows modifying the kinematic attribute at runtime? Thanks in advance!
